schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoader.java

```diff
@@ -60,4 +70,257 @@ public static void main(String... args) {
+   * @param createCoordinatorTable create coordinator table or not.
+   */
+  public static void load(
+      Properties configProperties,
+      @Nullable String serializedSchemaJson,
+      Map<String, String> options,
+      boolean createCoordinatorTable)
+      throws SchemaLoaderException {
+    Either<Path, Properties> config = new Right<>(configProperties);
+    Either<Path, String> schema = new Right<>(serializedSchemaJson);
+    load(config, schema, options, createCoordinatorTable);
+  }
+
+  /**
+   * Creates tables defined in the schema.
+   *
+   * @param configFilePath path to Scalar DB config file.
+   * @param serializedSchemaJson serialized json string schema.
+   * @param options specific options for creating tables.
+   * @param createCoordinatorTable create coordinator table or not.
+   */
+  public static void load(
+      Path configFilePath,
+      @Nullable String serializedSchemaJson,
+      Map<String, String> options,
+      boolean createCoordinatorTable)
+      throws SchemaLoaderException {
+    Either<Path, Properties> config = new Left<>(configFilePath);
+    Either<Path, String> schema = new Right<>(serializedSchemaJson);
+    load(config, schema, options, createCoordinatorTable);
+  }
+
+  /**
+   * Creates tables defined in the schema file.
+   *
+   * @param config Scalar DB config.
+   * @param schema schema definition.
+   * @param options specific options for creating tables.
+   * @param createCoordinatorTable create coordinator table or not.
+   */
+  public static void load(
+      Either<Path, Properties> config,
+      Either<Path, String> schema,
+      Map<String, String> options,
+      boolean createCoordinatorTable)
+      throws SchemaLoaderException {
+    SchemaOperator operator = getSchemaOperator(config);
+
+    // Create tables
+    try {
+      if (schema instanceof Left) {
+        Left<Path, String> schemaFilePath = (Left<Path, String>) schema;
+        if (schemaFilePath.getValue() != null) {
+          operator.createTables(schemaFilePath.getValue(), options);
+        }
+      } else {
+        Right<Path, String> serializedSchemaJson = (Right<Path, String>) schema;
+        if (serializedSchemaJson.getValue() != null) {
+          operator.createTables(serializedSchemaJson.getValue(), options);
+        }
+      }
+
+      if (createCoordinatorTable) {
+        operator.createCoordinatorTable(options);
+      }
+
+    } catch (SchemaOperatorException e) {
+      throw new SchemaLoaderException("Creating tables failed.", e);
+    } finally {
+      operator.close();
+    }
+  }
+
+  /**
+   * Delete tables defined in the schema file.
+   *
+   * @param configProperties Scalar DB config properties.
+   * @param schemaFilePath path to schema json file.
+   * @param options specific options for deleting tables.
+   * @param deleteCoordinatorTable delete coordinator table or not.
+   */
+  public static void unload(
+      Properties configProperties,
+      Path schemaFilePath,
```

Comment:
We also need `@Nullable` for the `unload()` methods, right?

Reply:
Yes, I missed them. Add it in [commit](https://github.com/scalar-labs/scalardb/pull/366/commits/0c8f3fe47b504108058a0f419a42bb8a6808f80d)